mne/tests/test_bem.py

```diff
@@ -333,4 +338,22 @@ def test_fit_sphere_to_headshape():
     assert_raises(TypeError, fit_sphere_to_headshape, 1, units='m')
 
 
+@requires_freesurfer
+@testing.requires_testing_data
+def test_make_flash_bem():
+    """Test computing bem from flash images."""
+    import matplotlib.pyplot as plt
+    flash_path = op.join(subjects_dir, 'sample', 'mri', 'flash')
+    # This function deletes some files at the end.
+    make_flash_bem('sample', overwrite=True, subjects_dir=subjects_dir,
+                   flash_path=flash_path)
+    plt.close('all')
+    inner_skull = op.join(subjects_dir, 'sample', 'bem', 'inner_skull.surf')
+    outer_skull = op.join(subjects_dir, 'sample', 'bem', 'outer_skull.surf')
+    outer_skin = op.join(subjects_dir, 'sample', 'bem', 'outer_skin.surf')
```

Comment:
files in the `MNE-testing-data` folder should not be modified, so +1 on doing something with a temporary directory like:

1. Copy only the necessary input files to temp dir
2. Run code
3. Make sure output files are now present